### PR TITLE
HTML, CSS & Git Basics

### PR DESCRIPTION
**Pull request by dnlklbv**
Completed task "HTML, CSS & Git Basics"

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{go,py,rb,php}]
```
Comment (quantum-noise) [inline .editorconfig]: There are no such files in the project =)

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,17 @@
+; http://editorconfig.org
```
Comment (quantum-noise) [inline .editorconfig]: Bonus for `.editorconfig` usage

File: assets/style/style.css [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    url("../fonts/lato/latobold.ttf") format("ttf");
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.row {
+  width: 920px;
```
Comment (quantum-noise) [inline assets/style/style.css]: Should be 1024 according to task requirements.

Comment (dnlklbv) [reply assets/style/style.css]: There is the only requirement for content to be not wider than 1024px.
[Link to requirements](https://github.com/rolling-scopes-school/tasks/blob/2018-Q3/tasks/codejam-cv.md#%D1%82%D1%80%D0%B5%D0%B1%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F-%D0%BA-htmlcss-%D1%80%D0%B0%D0%B7%D0%BC%D0%B5%D1%82%D0%BA%D0%B5)

**Pull request merged**